src/term/mod.rs

```diff
@@ -737,10 +742,20 @@ impl VisualBell {
         }
     }
 
+    pub fn run_bell_command(&self) {
```

Comment:
Another small naming note, would simply `run`, `spawn` or `start` be better here? `..._bell_command` is again somewhat redundant.